s/features/chat/components/xiome-chat/styles/chat-history.css.ts

```diff
@@ -2,10 +2,56 @@
 import {css} from "../../../../../framework/component.js"
 export default css`
 
-.history {}
+.history {
+	overflow-y: scroll;
+	display: flex;
+	text-align: center;
+	flex-direction: column-reverse;
+	height: 300px; /*Delete height values when the chat element gets styled*/
+}
 
-.history ol {}
+.history ol {
+	text-align: left;
+}
 
-.history li {}
+.history li {
+	border-radius: 10px;
```

Comment:
- please replace `px` units with `em` units.  
  using em's everywhere will allow a user to customize the whole component's scale (via `font-size`).  
  sometimes it's okay to use `px` in rare cases, like `border-width`.